bitnami/common/templates/_capabilities.tpl

```diff
@@ -26,6 +26,17 @@ Return the appropriate apiVersion for policy.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the appropriate apiVersion for networkpolicy.
+*/}}
+{{- define "common.capabilities.networkPolicy.apiVersion" -}}
+{{- if semverCompare "<1.7-0" (include "common.capabilities.kubeVersion" .) -}}
+{{- print "policy/v1beta1" -}}
```

Comment:
Shouldn't be `extensions/v1beta1` [deprecation-guide/#v1-16](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-16)?

Reply:
Ups you're right